### PR TITLE
feat(become-author-advertisement): created advertisemet component, changed layout become author page

### PR DESCRIPTION
**Pull request by DamianBrzezinskiHoA**
Appended:
-become author advertisement component
Modified:
-become author page

### CODE REVIEW

File: libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<al-card alGradientCard>
+  <div alCardContent>
+    <div class="flex flex-col md:flex-row lg:flex-col">
```
Comment (jakNatonek) [inline libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html:3]: Hmmm maybe container queries is better to use in this case ?
see https://github.com/tailwindlabs/tailwindcss-container-queries

Comment (DamianBrzezinskiHoA) [reply libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html]: I hadn't known container queries. Both solutions works properly but I decided to use tailwind container queries. Thank you

File: libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          Do you have an idea for an article? Would you like to start
+          contributing and creating the Angular Community with us?
+        </p>
+      </div>
+      <div
+        class="@xl:justify-end my-2 flex w-full flex-row items-center justify-center"
+      >
+        <a
+          class="bg-al-pink flex h-[46px] w-full items-center justify-center rounded-lg md:w-4/5 lg:w-full"
+          href="https://form.typeform.com/to/Zzf1Girt?typeform-source=angular.love"
```
Comment (DDonochVA) [inline libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html:18]: add `aria-label`

Comment (jakNatonek) [inline libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html:18]: i am not sure about this href magic string, don't we want to have this configurable?

Comment (DamianBrzezinskiHoA) [reply libs/blog/become-author/feature-become-author-page/src/lib/components/become-author-advertisement/become-author-advertisement.component.html:18]: No, AFAIK we only support english version. If this statement is true, we do not need any variable

**Review by DDonochVA — APPROVED** [review]

**Pull request merged**